examples/setup_notebook_colab.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Make a temporary download folder
+TEMP_DOWNLOAD_DIR=$(mktemp -d)
+TEMP_DATASET_DIR=$(mktemp -d)
+
+# Download sample zarr
+echo "Downloading sample zarr dataset..."
+wget https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/prediction/v1.1/sample.tar \
+    -q --show-progress -P $TEMP_DOWNLOAD_DIR
+
+mkdir -p $TEMP_DATASET_DIR/scenes
+tar xf $TEMP_DOWNLOAD_DIR/sample.tar -C $TEMP_DATASET_DIR/scenes
+
+# Download semantic map
+echo "Downloading semantic map..."
+wget https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/prediction/v1.1/semantic_map.tar \
+    -q --show-progress -P $TEMP_DOWNLOAD_DIR
+mkdir -p $TEMP_DATASET_DIR/semantic_map
+tar xf $TEMP_DOWNLOAD_DIR/semantic_map.tar -C $TEMP_DATASET_DIR/semantic_map
```

Comment:
can we move `meta.json` in `TEMP_DATASET_DIR`?

Reply:
I totally forgot about it, will fix it.

Reply:
Fixed